scripts/commands.js

```diff
@@ -38,6 +39,14 @@ program
   .command('versions')
   .action(versions)
 
+program
+  .command('header_deps')
+  .option('--target_os <target_os>', 'target OS')
```

Comment:
I guess it needs these to check the correct output dir?

Reply:
right